source/state_representation/src/robot/JointVelocities.cpp

```diff
@@ -173,7 +180,7 @@ std::ostream& operator<<(std::ostream& os, const JointVelocities& velocities) {
   } else {
     os << velocities.get_name() << " JointVelocities" << std::endl;
     os << "names: [";
-    for (auto& n : velocities.get_names()) { os << n << ", "; }
+    for (auto& n: velocities.get_names()) { os << n << ", "; }
```

Comment:
I don't know why or where this is style change is coming from for range based iterations 🤨 it's interesting, I always saw it with `a : b` before and don't recall ever changing (or even finding) any specific setting in the XML that regulates this. Maybe it's the new CLion version default? 

Anyway since it seems to be the new consistent behaviour (also on my CLion 2021 client) so that's just the new style rule I guess 😆

Reply:
self evolving style guide is an interesting feature :D